image/image.html

```diff
@@ -3,12 +3,54 @@
     category: 'output',
     color: '#a6bbcf',
     defaults: {
+      active: {value:true},
       name: { value: "" }
     },
     inputs: 1,
     outputs: 0,
     icon: "watch.png",
     align: 'right',
+    button: {
+        toggle: "active",
```

Comment:
Lets keep the indentation as 2 spaces. At least VSCode's `Format document` will fix these automatically

Reply:
Start using VSCode is on my TODO list.  But the list is growing ;-)